src/angular/tooltip/tooltip.md

```diff
@@ -47,11 +47,12 @@ To completely disable a tooltip, set `disabled` on `sbb-tooltip` or `sbbTooltipD
 
 ### Accessibility
 
-Elements with the `sbbTooltip` will add an `aria-describedby` label that provides a reference
-to an element containing the tooltip's message. Depending on the used variant this is either a
-hidden element with a copy of the message or a reference to the tooltip element itself.
-
-If a tooltip will only be shown manually via click, keypress, etc., then extra care should be taken
-such that the action behaves similarly for screen-reader users. One possible approach would be
-to use the `LiveAnnouncer` from the `cdk/a11y` package to announce the tooltip content on such
-an interaction.
+`SbbTooltip` adds an `aria-describedby` description that provides a reference
```

Comment:
This is not the same, since I changed part of the implementation. aria-describedby either points to a copy (string usage) or the element itself (template usage).